phd-tests/framework/src/disk/crucible.rs

```diff
@@ -0,0 +1,275 @@
+//! Abstractions for Crucible-backed disks.
```

Comment:
Thanks for the pointer! I had a look and suspect `dsc` is too sophisticated for PHD's needs in this PR but may be useful later. If we switched to it, I think PHD would still have to manage all the same command-line arguments and process executions that it does today; it would just have `dsc` be the target of these calls instead of `crucible-downstairs`.

Down the line, though, I think it might be neat to use `dsc`'s server start/stop commands to simulate Crucible failures in more interesting ways than "kill the downstairs process," which I think is about the extent of PHD's capacity for subtlety in this PR.